pom.xml

```diff
@@ -13,6 +13,13 @@
 	<description>A kafka health check for spring boot actuator</description>
 	<url>https://github.com/deviceinsight/kafka-health-check</url>
 
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-dependencies</artifactId>
+		<version>2.3.8.RELEASE</version>
+		<relativePath />
+	</parent>
```

Comment:
@pvorb Any reservations against this approach? We use spring-boot-dependencies in our internal projects to keep the dependency versions unified across all projects.

Reply:
@whackenb @ezienecker I think the approach to use "spring-boot-dependencies" as a parent is a bit misleading. This is typically done with "spring-boot-starter-parent" to also inherit plugin configurations and so on. I don't think this is the goal here, since you inherit from -dependencies rather than -starter-parent.

I think the typical way to do this would be to import "spring-boot-dependencies" under `<dependencyManagement />`:

~~~ xml
<dependencyManagement>
     <dependencies>
        <dependency>
            <!-- Import dependency management from Spring Boot -->
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-dependencies</artifactId>
            <version>2.3.8.RELEASE</version>
            <type>pom</type>
            <scope>import</scope>
        </dependency>
    </dependencies>
</dependencyManagement>
~~~

Reply:
* But it's not really *wrong*, so it's up to you.

Reply:
Yes, choosing dependencies over starter-parent was on purpose, to avoid adding unneeded plugin configuration to the project.

The import approach has limitations afaik: When you import the dependencies, the version properties are not imported. You loose the option to override versions without duplicating the dependency management entry or use version property defined in the Spring Boot dependencies.

But I wouldn't mind to inheriting from `spring-boot-parent-starter`.

Reply:
That's a valid remark. In that case, inheriting from "spring-boot-dependencies" is also fine for me.